pubsub/google/cloud/pubsub_v1/subscriber/message.py

```diff
@@ -88,6 +89,10 @@ def __init__(self, message, ack_id, request_queue, autolease=True):
             autolease (bool): An optional flag determining whether a new Message
                 instance should automatically lease itself upon creation.
                 Defaults to :data:`True`.
+
+                .. note::
+                    .. deprecated:: 0.43.1
```

Comment:
I would be inclined not to add deprecations in a patch release.

Reply:
That was exactly my question in the email thread. :)

I suppose it would be better to do a _minor_ version bump for the deprecation release, i.e. `0.44.0`?

Reply:
That is how I would do it, yes.